samples/drivers/watchdog/boards/stm32h7_wwdg.overlay

```diff
@@ -11,10 +11,6 @@
 	};
 };
 
-&rcc {
-	d2ppre1 = <16>;
-};
-
```

Comment:
Can you please state in the commit message why the RCC setting was wrong? You added it a while ago in commit 838cd28195631e039b569f39cdc10cf1c839807f saying it is required.

Reply:
You're right
 even if at that time, the sysclock was much lower (typ. 96MHz) for the stm32h7xx, (now it is 480) there is no need to change it.